CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # Mattermost Mobile Apps Changelog
 
+## 1.29.0 Release
+- Release Date: March 16, 2020
+- Server Versions Supported: Server v5.9+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
+
+### Compatibility
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).
+ - iPhone 5s devices and later with iOS 11+ is required.
+ 
+### Improvements
+ - Improved the experience where the app was unresponsive for a period of time after clearing cache or on initial login.
+ 
+### Bug Fixes
+ - Fixed an issue where downloaded files on Android had the words `download successful` appended to their filenames, preventing the file from being opened until it was renamed in the file manager.
+ - Fixed an issue on Android where users could not swipe to close sidebar unless the gesture was initiated outside of the sidebar.
+ - Fixed an issue where channels drawers were partially shown with orientation change on iOS RN61.
+ - Fixed an issue where the message box obstructed the bottom part of the message when opened from the notification banner.
+ - Fixed an issue where switching teams showed the center channel from the old team until the new team's channel data got loaded.
+ - Fixed an issue on Android where inline custom emojis did not display in portrait mode.
+ - Fixed an issue where a user's was set as online when replying to a message from a push notification.
```

Comment:
this only works in combination with servers 5.20+

Reply:
Does this need a compatibility note?

Reply:
So basically the "bug" is fixed with a combination of the app with the server, on previous servers it will still set the user as online. So I would say yes!

Reply:
1. It only applies when the user replies to a push notification, the server code was setting the user as online, a flag to avoid that from happening was added in 5.20
2. Server code was introduced in 5.20 we only applied it in the mobile app on this release
3. Not sure if it is an incompatibility, it just won’t set the server as online if the server is 5.20 or grater as the flag did not exist before 5.20

Reply:
Ok, added a note.